### DISCUSSION TITLE
NOP-In not supported.

### DISCUSSION THREAD

**Discussion by gobiursus** [ipxe/ipxe#1225] [General]
Hi,
Can we add support to NOP-In on iSCSI? My connection to my iSCSI volume won't last more than 30 seconds.

**Comment by ThatProgrammerr** [comment]
Is this in relation to the error of 'unsupported NOP-In with TTT 000006e8'. I've been trying to PXE boot into my iSCSI lun but I always hit a brick wall when this happens. I'm not sure the solution.
